doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+  * [5.1.3 Network Layering and Concepts](#5.1.3)
+
 
 <a name="5.1"></a>
 ## 5.1 Networking and Fabric Approach
 
-> _**Editor's Note:** The purpose in the subsections is to provide initial high-level content related to networking across CNTT, so as to provide general direction and help coordinate current independent networking development activities, as well as to satisfy the networking MVP deliverables for the Baldy release. The following content addresses the overall CNTT networking solution, and it is anticipated this content will be expanded and be relocated where appropriate._
+The networking within a cloud infrastructure is an area where there is significant variability across implementations. Leaf-Spine topology for large datacenters is well established, however after physical switch interconnection topology choices there are countless decisions an Operator needs to make. Differences arise from many aspects, for example, is the solution layer-2 or layer-3; is the routing static or dynamic; what mechanism is used for encapsulation; what mechanism is used for isolation; does it support SR-IOV; does it support DPDK; does it employ SmartNICs; does it employ distributed control or a centralized control driving a programmable networking fabric; are the network switches selected and provisioned by a separate Network Operations group or by a Cloud Infrastructure Operations group; and the list continues. The multitude of permutations enable cloud infrastructure architects (Operators and Suppliers) to design (or procure) a fabric/networking solution that's optimized for their needs, whether their needs are minimal, very extensive or somewhere in between.
+
+We need a strategy that affords Operators the performance, flexibility, availability, maintainability and scalability their business requires, yet doesn't require OPNFV to design, manage and test prohibitive numbers of networking solutions.
+ 
 
-The basic approach to producing the initial deliverables is to:
-A) Summarize the overall CNTT Networking and Fabric Strategy
-B) Document the initial list of Objectives and Requirements
-C) Document enough initial logistical details for contributors to create coherent content
+<a name="5.1.1"></a>
+### 5.1.1 Networking Scope
+According to the ETSI NFV Model, Networking alongside Compute and Storage, is an integral part of the Network Function Virtualisation Infrastructure (NFVI). The general function of Networking in the NFV context is to provide the connectivity between various virtualised and non-virtualised resources required for a delivery of a Network Service. Such a connectivity may manifest itself as a virtualised network between VMs and/or Containers (e.g. overlay networks managed by SDN Controllers, and/or programmable networking fabrics that provide such connectivity natively) or as an integration into the Infrastructure Hardware level for offloading some of the Network Functions into this level (accelerated networking functions). 
```

Comment:
* Lots of NFV references in the first sentence. Do we need to change this to Cloud Infra? Are we going to be able to remain agnostic/generic, and simultaneously tightly couple ourselves directly to ETSI NFV?

* "Such **a** connectivity may..." Remove the "a"

* "e.g. overlay networks managed by SDN Controllers, and/or programmable networking fabrics _that provide such connectivity natively_" - Can we clarify what we mean by the fabrics providing the connectivity "natively"? 

*  "as an integration into the Infrastructure Hardware level for offloading some of the Network Functions into this level" - Need to make this a bit clearer. Which Infra h/w level? Offloading some of the NFs into this level?

Reply:
I think we can not eradicate all references to ETSI NFV, since that is a lot of the vital foundation. We can talk about Cloud Infrastructure, when we have defined it and in our models, but not in the references to ETSI NFV.

Reply:
>
> I think we cannot eradicate all references to ETSI NFV...

I agree. However, when reading 5.1.1, it seems more like we're tightly coupled to it, than we're referencing it.

Reply:
We need to describe the networking aspects for the cloud infrastructure here, not the NFVI, if we use the wording of the rest of the document. E.g. the architecture diagram 5-1-4 is not describing the ETSI NFV reference architecture, but a layered cloud infrastructure - which has many similarities, but also many differences to ETSI. There are lots of reference points and functional blocks which are unknown to ETSI. Thus it should be a diagram of the cloud infrastructure, etc.
Btw. Does the HW resource pool need to be hardware? Could it be a cloud?

Reply:
Agree on that we likely should reword these descriptions a bit and by that more augment our Cloud Infrastructure Networking Model on top of the base ETSI NFV model at least ensuring that they do not contradict each other.
I would though suggest that this is a slightly larger task that likely comes after Baldy.

@ulikleber Yes, the HW resource pool needs to be HW somewhere in the foundation, but it should not need to be reachable or visible for the layers above the HW Infrastructure Layer according to these chapters. By that also follows that the HW Infrastructure could be supplied by a Public or Private Cloud Provider that manages its own HW resource pools completely out of reach from the SW Virtualization Layers and VNFs/CNFs.